packages/halo-theme/src/custom-elements/ef-toggle.less

```diff
@@ -54,10 +54,10 @@
     &:hover:not([readonly]) {
       color: @input-hover-text-color;
       [part=toggle] {
-        border-color: @scheme-color-primary;
+        border-color: @color-persian-blue;
```

Comment:
We do not use colors directly for custom elements. Instead, you can either create a new variables in `variables.less` or better to use already existing variables with similar meaning (e.g. from button)

Reply:
Thanks, updated.